common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -434,33 +473,36 @@
             1
         ],
         "import/no-named-as-default-member": [
-            1
+            "off"
```

Comment:
I'm not sure about no-named-as-default-member but maybe good to at least warn?

Reply:
Disabled because TypeScript already checks this. <https://typescript-eslint.io/docs/linting/troubleshooting/#eslint-plugin-import>